content/docs/user-guide/checkpoints.md

```diff
@@ -0,0 +1,423 @@
+```
+
+You'll see output similar to this in your terminal while the training process is
+going on.
+
+```dvc
+Checkpoint experiment iteration 'd50c724'.
+Updating lock file 'dvc.lock'
+Checkpoint experiment iteration '29491a9'.
+Updating lock file 'dvc.lock'
+Checkpoint experiment iteration 'b3de55f'.
+Updating lock file 'dvc.lock'
+Checkpoint experiment iteration 'c4a46af'.
+Updating lock file 'dvc.lock'
+Checkpoint experiment iteration 'daf204c'.
+Updating lock file 'dvc.lock'
+Checkpoint experiment iteration 'bdb975c'.
+Updating lock file 'dvc.lock'
+Checkpoint experiment iteration '77dc46c'.
+Updating lock file 'dvc.lock'
+```
+
+After a few epochs have completed, stop terminate the training process with
+`Ctrl + C`. Now it's time to take a look at the metrics we're working with.
+
+_If you don't have a number of training epochs defined and you don't terminate
+the process, the experiment will run for 10 minutes._
+
+## Viewing checkpoints
+
+You can see a table of your experiments and checkpoints in the terminal by
+running:
+
+```bash
+dvc exp show
+```
+
+The accuracy on our current checkpoints will be high because we've already run
+them twice and training always picks up from the last checkpoint.
+
+```dvc
+┏━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━┳━━━━━━┳━━━━━━━━━┳━━━━━━━━┳━━━━━━━━┳━━━━━━━━┳━━━━━━━━━━━━━━┓
+┃ Experiment            ┃ Created  ┃ step ┃    loss ┃    acc ┃ seed   ┃ lr     ┃ weight_decay ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━╇━━━━━━╇━━━━━━━━━╇━━━━━━━━╇━━━━━━━━╇━━━━━━━━╇━━━━━━━━━━━━━━┩
+│ workspace             │ -        │    7 │  0.3096 │ 0.9092 │ 473987 │ 0.0001 │ 0            │
+│ main                  │ 04:49 PM │    - │       - │      - │ 473987 │ 0.001  │ 0            │
+│ │ ╓ exp-4de2a         │ 05:03 PM │    7 │  0.3096 │ 0.9092 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ d50c724           │ 05:03 PM │    6 │ 0.33246 │ 0.9044 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ 29491a9           │ 05:02 PM │    5 │ 0.36601 │ 0.8943 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ b3de55f           │ 05:02 PM │    4 │ 0.41666 │ 0.8777 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ c4a46af           │ 05:02 PM │    3 │ 0.50835 │ 0.8538 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ daf204c           │ 05:02 PM │    2 │ 0.72421 │ 0.8284 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ bdb975c           │ 05:02 PM │    1 │  1.2537 │ 0.7738 │ 473987 │ 0.0001 │ 0            │
+│ ├─╨ 77dc46c           │ 05:01 PM │    0 │  1.9428 │ 0.5715 │ 473987 │ 0.0001 │ 0            │
+```
+
+## Starting from an existing checkpoint
+
+Since you have all of these different checkpoints, you might want to resume
+training from a particular one. For example, maybe your accuracy started
+decreasing at a certain checkpoint and you want to make some changes to fix
+that. You can start training from any existing checkpoint with the following
+command.
+
+If you want to start a new experiment based on an existing checkpoint, you can
+run:
+
+```bash
+dvc exp apply b3de55f && dvc exp run
```

Comment:
`dvc exp apply b3de55f && dvc exp run` - this action is not very useful. Users will get the same result.
We should do some code modification (like increase learning rate) before `dvc exp run`.